test/test-ParametersParser.cc

```diff
@@ -271,6 +271,28 @@ TEST_CASE("Parentheses in parameter example ", "[parameters][issue][#109]")
     REQUIRE(parameters.node[0].description == "test");
 }
 
+TEST_CASE("Parse parameters when it has parameter of both old and new syntax", "[parameter]")
+{
+    mdp::ByteBuffer source = \
+    "+ Parameters\n"\
+    "    + id (optional, string) ... Hello\n"\
+    "    + percent_off: 25 (required, number)";
+
+    ParseResult<Parameters> parameters;
+    SectionParserHelper<Parameters, ParametersParser>::parse(source, ParametersSectionType, parameters);
+
+    REQUIRE(parameters.report.error.code == Error::OK);
+    REQUIRE(parameters.report.warnings.empty());
+
+    REQUIRE(parameters.node.size() == 2);
+    REQUIRE(parameters.node[0].name == "id");
+    REQUIRE(parameters.node[0].type == "string");
+    REQUIRE(parameters.node[0].description == "Hello");
+    REQUIRE(parameters.node[1].name == "percent_off");
+    REQUIRE(parameters.node[1].type == "number");
+    REQUIRE(parameters.node[1].exampleValue == "25");
+}
```

Comment:
parameter has `+ percent_off: 25 (required, number)"` Is there in node somewhere `required` holded? If yes, I mean it should be checked.

Sure if it is already tested in other unit test, we need no it